public-ui/src/app/utils/spinner.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+export const spinnerCss = `
```

Comment:
This is fine for now, but may change when we start aligning styles more with workbench. May be good to separate styles used for the different spinners if we split them into separate consts

Reply:
I have seperated styles. Ll take a look at workbench styles.

Reply:
Yeah, I'm going to try to work on styles some next week. Converting the dots CSS here to objects looks pretty straightforward, not sure about some of the spinner styles like `@keyframes` and `a:link,a:visited`.